frontend_lib/test/index.js

```diff
@@ -15,4 +15,3 @@ export {
   user,
   workspace
 }
-
```

Comment:
keep the last blank line at the end

Reply:
In fact there were two blank lines at the end, I removed only one.